src/stylesheets/main.css

```diff
@@ -55,7 +55,6 @@ main.container {
 
 .navbar {
   --bs-navbar-padding-y: 0;
-  box-shadow: 0px 4px 4px var(--calitp-cyan-1);

```

Comment:
I think we do want this `box-shadow`. The Figma screen shows one:

![image](https://github.com/cal-itp/calitp.org/assets/25497886/bc4bf2e6-aa53-4fb0-b1d5-1ba7ba30d634)


Staging vs. this PR
![image](https://github.com/cal-itp/calitp.org/assets/25497886/035e382c-d6fc-4159-91f6-ed192f41b4a2)

![image](https://github.com/cal-itp/calitp.org/assets/25497886/39ddc578-ec52-43d4-af6d-c0257532ff35)